tensorboard/program.py

```diff
@@ -208,6 +207,8 @@ def configure(self, argv=('',), **kwargs):
 
     with argparse_util.allow_missing_subcommand():
       flags = base_parser.parse_args(argv[1:])  # Strip binary name from argv.
+    if getattr(flags, _SUBCOMMAND_FLAG, None) is None:
```

Comment:
Maybe worth adding a TODO/note here so we can revert after we drop support for old python?

Reply:
Done.